src/aks-preview/azext_aks_preview/__init__.py

```diff
@@ -36,6 +36,17 @@ def __init__(self, cli_ctx=None):
     def load_command_table(self, args):
         super().load_command_table(args)
```

Comment:
Thank you for assisting with the change. However, I believe that with cli-extensions/aks-preview installed, users can access the `az aks safeguards` command group even without the change here, since it's already available in the official azure-cli, right?

Reply:
You are right, thank you-- I've removed this part